frontend/packages/ceph-storage-plugin/integration-tests/mocks/storage-class.ts

```diff
@@ -1,10 +1,16 @@
-export const storageClass = {
+export const mockNoProvisionerSC = {
```

Comment:
```suggestion
export const NoProvisionerSC = {
```
No need to add mock in front as it's all inside the mock folder.

Reply:
When I see `mockNoProvisionerSC` somewhere I can make out that it is a mock storage class.

Reply:
Let's follow the pattern that we already have set.

Reply:
Renamed the test classes to be prefixed with "test" .